bigquery_etl/run_query.py

```diff
@@ -0,0 +1,107 @@
+"""
```

Comment:
This was moved here from `scripts/run_query`

Reply:
`scripts/run_query` should be changed to call this.